acceptance/api_v1_applications_test.go

```diff
@@ -218,6 +218,7 @@ var _ = Describe("Apps API Application Endpoints", func() {
 
 			// make the request
 			request, err := http.NewRequest("POST", url, body)
+			request.SetBasicAuth("me", "change")
```

Comment:
maybe instead of relying on the defaults in tests, we can (indirectly) test that the installation parameters work by specifying some credentials when we install Epinio in the SynchronizedBeforeSuite block.

Reply:
I did so in a separate commit.